.gitignore

```diff
@@ -37,13 +68,9 @@ coverage.xml
 node_modules/
 bower_components/
 .grunt/
+vendor/
+static/vendor/
 src/vendor/
-src/static/fonts/
-
-# Jekyll #
-##########
-_site/
-
-# Ruby #
-########
-Gemfile.lock
```

Comment:
I've had issues in the past on this repo when I came back to it after a while, but my `Gemfile.lock` file was old and out of date. It was keeping me pegged to several versions behind on gems.